main/pipeline/pipeline.cc

```diff
@@ -352,7 +352,11 @@ core::StrictLevel decideStrictLevel(const core::GlobalState &gs, const core::Fil
     auto &fileData = file.data(gs);
 
     core::StrictLevel level;
-    auto fnd = opts.strictnessOverrides.find(string(fileData.path()));
+    string filePath = string(fileData.path());
+    if (filePath.find("/") != 0 && filePath.find("./") != 0) {
```

Comment:
This needs to be present here because otherwise the use-case of
```bash
$ sorbet --typed-ignore=foo.yaml bar.rb
```
would fail, because all the paths from `foo.yaml` would have been normalized to include `./`.